requests-for-comments/2023-17-produchog.md

```diff
@@ -0,0 +1,92 @@
+
+Most recent issue for context + feedback: [https://mailchi.mp/posthog/hogmail-15794324](https://mailchi.mp/posthog/hogmail-15794324)
+
+## Where we are now
+
+- Our subscriber list has grown 49% since April – 662 to 987 as of writing.
+- Open rates have increased from ~43% at launch to ~47% to 50% now. Based on my research, this is above average but not market-leading for a newsletter of our type.
+- Each issue is split into three sections: new PostHog articles, curated articles with commentary, misc. interesting reads.
+- Most signups come via CTAs on the website and promotion in The Array
+- At the current rate of growth, we’d reach 1,500 subscribers by the end of 2023
+
+## Where we want to be
+
+- 3,000 subs by end of 2023 – a 200% increase on today
+- People sharing and recommending newsletter on social media
+- Be a newsletter that’s perceived as a desirable source of promotion for content creators
+
+## Proposal: Rebrand to ‘ProductHog’ and launch on Substack
+
+- The current branding was a simple way to get started, but is limiting.
+- ‘ProductHog’ implies what the newsletter is about, while still drawing a connection between it and PostHog.
+- Actions: create website for ProductHog (we own the domain) and launch dedicated social channels.
+
+### Why Substack?
+
+Currently the newsletter is on Mailchimp, but we’ve recently migrated other email ops to [Customer.io](http://Cusomter.io). 
+
+I’m proposing Substack because:
+
+- It’s a popular platform people know and like
+- It would make this feel distinct from other “marketing emails”
+- Allows us to tap into the Substack network effects to aid growth
+
+The downsides of Substack:
+
+- We’ll have less control over branding and appearance
+- **Important:** Substack’s [official embeddable signup form](https://support.substack.com/hc/en-us/articles/360041759232-Can-I-embed-a-signup-form-for-my-publication-) is very limited – it’s not customisable and there’s no public API or any sign they’re going to support one. [Viable workarounds](https://docs.substackapi.com/) may exist, but this is a major downside. The default Substack signup box (I believe) redirects people to Substack’s website as a result. Is this an acceptable trade-off for Substack’s network effects?
+
+If not Substack, we’ll move to [Customer.io](http://Customer.io) as originally planned.
+
+### Who is ProductHog for?
+
+As is currently the case, it’ll be closely aligned to our ICP and target in order of importance:
+
+- CEOs and founders
+- Product engineers
+- Product managers and product owners
+- Growth hackers and product marketers
+
+### Who is ProductHog NOT for?
+
+- Indie hackers (they may find it useful, but it won’t be tailored to their interests)
+- Marketing teams (even when we discuss marketing adjacent topics)
+- Traditional “unengaged” engineers
+- Data scientists
+
+## How to 10x ProductHog
+
+### Make each issue more thematic
```

Comment:
Although it is a literally a magazine, I think [Increment](https://increment.com/) is a gold standard for this (with original content).